--- a/src/main/java/eu/kartoffelquadrat/ls/accountmanager/controller/AccountForm.java
+++ b/src/main/java/eu/kartoffelquadrat/ls/accountmanager/controller/AccountForm.java
@@ -80,7 +80,7 @@
 
     public static boolean validatePasswordString(String password)
     {
-        return Pattern.compile("(?=.*[0-9])(?=.*[a-z])(?=.*[A-Z])(?=.*[^A-Za-z0-9]).{8,32}").matcher(password).find();
+        return password != null && !password.trim().isEmpty();
     }
 
     public static boolean validateColourString(String colourString)
